Catalog/DataAccessors/Comments.cs

```diff
@@ -0,0 +1,116 @@
+﻿using Catalog.Utils;
+using Common.Interfaces;
+using Common.Models.Comments;
+using Dapper;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Catalog.DataAccessors
+{
+    /// <summary>
+    /// Comments Dapper Accessor
+    /// </summary>
+    public class Comments : IDataAccessor<Comment>
+    {
+        private string query;
```

Comment:
I see no reason to move this variable here. This can make some hard-to-find errors, in case query won't be replaced with the proper one, and you will try to execute the previous version of the query.
- [ ] Move query variable to each method separately to avoid common dependency between methods with the different purpose.